tests.py

```diff
@@ -8,12 +8,28 @@
 import os
 import unittest
 from subprocess import Popen, PIPE
+from ipc.loader import load_plugin
 
 
 RULES_PATH = os.path.abspath(os.path.join('.', 'sample_rules'))
 
+def send(signal, value):
+    "Function is overriden by plugin"
+
+def receive():
+    "Function is overriden by plugin"
+
+def format_ipc_input(data):
+    return [ tuple(elm.split(' = ')) for elm in data.split('\n') ]
+
+def format_ipc_output(data):
+    return (('', '') if data == '' else tuple(data.strip().split(' = ')))
```

Comment:
Any reason for this?

The way I see it, we shouldn't manipulate the output from the IPC message, but instead pass a correct expected IPC message to compare.